docs/dev/new_check_howto.md

```diff
@@ -274,6 +274,72 @@ The check is almost done. Let's add the final touches.
 
 ## Final touches
 
+### Enhance the configuration file
+
+Add a set of special words to the check `conf.yaml.example` configuration file in form of YAML comment strings. 
+Such words are referred to as *commands* and take the form `@command`. A command is valid only when the comment line containing it starts with a double `#` char:
+
+```
+## @command this is valid
+
+# @command this is not valid and will be ignored
+```
+
+Parameters in a configuration file follow these rules:
+
+* Placeholders should always follow this format: `<THIS_IS_A_PLACEHOLDER>`according to the documentation [contributing guidelines][16]: 
+* All required parameters are **not** commented by default but can contain placeholders
```

Comment:
Remove `can contain placeholders` in both sentences since there are no cases where it can not be that way.